src/components/Card.svelte

```diff
@@ -6,6 +6,7 @@
   import { ProjectThemes, ProjectThemesFa } from "../enums/themes.enum";
   import "../style/typography.css";
   import "../style/card-theme.css";
+  import {  ButtoDisplayoption  } from "../enums/button-selector.enum";
```

Comment:
Should be like this in every files
```suggestion
  import { ButtonDisplayOption } from "../enums/button-selector.enum";
```